tests/foreman/cli/test_setting.py

```diff
@@ -226,8 +226,17 @@ def test_positive_update_email_delivery_method_sendmail():
 
     :CaseImportance: Low
 
-    :CaseAutomation: NotAutomated
+    :CaseAutomation: Automated
     """
```

Comment:
Please add "CaseLevel, Steps and expectedresults"

Reply:
Steps and expectedresults are already there. They were hiding :grin:

Reply:
yes, I'll add the CaseLevel

Reply:
CaseLevel is already mentioned at the top of the file, so I think we no need to mention it as test case level.

Reply:
Okay.